### DISCUSSION TITLE
[field-creator] error message when using the 'field creator'

### DISCUSSION THREAD

**Discussion by Ln92-pixel** [tprouvot/Salesforce-Inspector-reloaded#640] [Q&A] [answered]
Hello,
I'm having trouble importing fields from 'field creator'

example : 
Contact,MasterRecordId,Lookup(Contact)

Invalid type Lookup(Contact)

Is it possible to import  lookup fields?

thanks

**Comment by tprouvot** [comment, marked as answer]
Hi @Ln92-pixel,
Currently the lookup fields are not supported but this info was missing in the [documentation](https://tprouvot.github.io/Salesforce-Inspector-reloaded/field-creator/) so I added it thanks to your question.
@SantiParris8 FYI

**Reply by Ln92-pixel** [reply]
thank you @tprouvot !  it is clear now
